docs/aws-professional/services.md

```diff
@@ -184,7 +184,7 @@ For an overview of Azure for AWS users, see [Introduction to Azure for AWS profe
 | [`eksctl`](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-eksctl.html) | [`az aks`](/cli/azure/aks) | Manage Azure Kubernetes Service using these Azure CLI commands. |
 | [AWS CloudShell](https://aws.amazon.com/cloudshell) | [Azure Cloud Shell](/azure/cloud-shell/overview) | Azure Cloud Shell is an interactive, authenticated, browser-accessible shell for managing Azure resources. It gives you the flexibility to choose the shell experience that best suits the way you work, either Bash or PowerShell. |
 | [OpsWorks (Chef-based)](https://aws.amazon.com/opsworks) | [Automation](https://azure.microsoft.com/services/automation) | Configures and operates applications of all shapes and sizes, and provides templates to create and manage a collection of resources. |
-| [CloudFormation](https://aws.amazon.com/cloudformation) | [Resource Manager](https://azure.microsoft.com/features/resource-manager) <br/><br/>[Bicep](/azure/azure-resource-manager/bicep/overview) <br/><br/>[VM extensions](/azure/virtual-machines/extensions/features-windows) <br/><br/>[Azure Automation](https://azure.microsoft.com/services/automation) | Provides a way for users to automate the manual, long-running, error-prone, and frequently repeated IT tasks.
+| [CloudFormation](https://aws.amazon.com/cloudformation) <br/><br/>[CDK](https://aws.amazon.com/cdk) | [Resource Manager](https://azure.microsoft.com/features/resource-manager) <br/><br/>[Bicep](/azure/azure-resource-manager/bicep/overview) <br/><br/>[VM extensions](/azure/virtual-machines/extensions/features-windows) <br/><br/>[Azure Automation](https://azure.microsoft.com/services/automation) | Provides a way for users to automate the manual, long-running, error-prone, and frequently repeated IT tasks.
```

Comment:
I don't think Azure has an analogous item to CDK. and Saying that it's bicep, VM extensions, and Azure Automation doesn't even really come close.  I don't think it's fair to AWS that we think these three services make up CDK.

You might instead find that CDK could be better compared to [azd](https://learn.microsoft.com/azure/developer/azure-developer-cli/) and [VTM](https://azure.github.io/Azure-Verified-Modules).

So, please change this to 
* have CDK be its own line item
* Spell CDK out
* Use AZD and VTM (spelled out) on the Azure side.